src/strategies/poap/index.ts

```diff
@@ -0,0 +1,70 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+export const author = 'greenealexander';
+export const version = '1.0.0';
+
+const POAP_API_ENDPOINT_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/poap-xyz/poap',
+  '100': 'https://api.thegraph.com/subgraphs/name/poap-xyz/poap-xdai'
+};
+// subgraph query in filter has max length of 500
+const MAX_ACCOUNTS_IN_QUERY = 500;
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  snapshot
```

Comment:
@gawainb Make sure you are using `snapshot` else the result of the proposal may keep changing.